README.md

```diff
@@ -4,25 +4,24 @@ building a basic website that allows users to add/remove books from a list by us
 ## Built With
 
 - HTML
+- CSS
 - javaScript
 
 ## Getting Started
 To get a local copy up and running follow these simple example steps:
 ## prerequisites
--must have git installed.
+- must have git installed.
 
```

Comment:
- [ ] **Required:** Kindly write more actionable steps that can be followed to run your project.